tests/README.md

```diff
@@ -13,15 +13,16 @@ Usage: ```python get_orbits.py```
 This saves orbits to `data/sample_orbits.csv`.
 
 #### `make_observations.py`:
-Produces a synthetic observations CSV file containing observations for each object in the `sample_orbits.csv` file. The observations have the following properties:
-- 4 observations each day (a "quad" tracklet) for 15 days from the epoch at which the orbit is defined (60 observations total).
-- Each set of 15 observations (in order of increasing time) is made by a different observatory: "500", "I11", "I41", "F51".
-- The first observation on any night is always an integer number of days from the reference epoch at which the orbit is defined. Observations 2, 3, and 4 in the same night, are set to be 30, 60, and 90 minutes from the time of the first observation, respectively.
+Produces 4 synthetic observations CSV files containing observations for each object in the `sample_orbits.csv` file. The observations have the following properties:
+- 4 observations each day (a "quad" tracklet) for 15 days for each observatory.
+- Each observatory ('500', 'I11', 'I41', 'F51') is split into its own unique dataset.
```

Comment:
Unlikely to cause a but, but might make more sense to collapse one of these to have multiple observatories in a single dataset so it resembles datasets like NSC?

Reply:
I think that's a great idea to be on the safe side. I'll rebase and then combine the F51 and I41 observations into a single dataset.